docs/mobile/concepts-commands-operations.mdx

```diff
@@ -178,7 +178,7 @@ This example demonstrates two different types of argument passing behavior. The
 -   `highlights.add-focus`
 -   `highlights.pulse`
 
-`tasks.identify` does not have any named arguments defined, so it will take arguments passed into it from it's current [context](https://docs.geocortex.com/webviewer/latest/admin-help/Default.htm#gwv/add-a-workflow-to-an-app.htm#use-the-menu-context-as-a-workflow-input). Since this chain is run on a map click, the context argument passed in has the shape
+`tasks.identify` does not have any named arguments defined, so it will take arguments passed into it from it's current [context](https://docs.geocortex.com/webviewer/latest/admin-help/Default.htm#gwv/add-a-workflow-to-an-app.htm#use-the-menu-context-as-a-workflow-input). Since this chain is run on a map click, the context argument passed in has the shape.
```

Comment:
I didn't have a period here because the sentence isn't complete; it's just interrupted by a code snippet. Perhaps a semicolon would be more appropiate?

Reply:
As discussed, changed to colon.